src/scaffold/zome/coordinator.rs

```diff
@@ -55,8 +55,20 @@ pub fn initial_lib_rs(dependencies: &Option<Vec<String>>) -> String {
             s
         }
     };
-
-    format!(
+    if simplified {
```

Comment:
Do we really need this? Are you concerned about them doing `hc scaffold zome` in an app where they have used the vanilla template, so after they have scaffolded an example hello world? If not, I don't see what effect this has on `hc scaffold hello-world` itself.

Reply:
Oh right, that was my old stuff before we went down the route of moving it all to the template.  I will remove.

Reply:
Fixed